src/shared/trim-and-dedent.ts

```diff
@@ -21,6 +36,8 @@ function dedent(str: string) {
   return str;
 }
 
+export type TrimAndDedentOptions = DedentOptions;
```

Comment:
Defining the TrimAndDedentOptions type as its own, so that we can define it as an intersection of `TrimOptions & DedentOptions` if `TrimOptions` are needed at some point.